README.md

```diff
@@ -56,14 +56,32 @@ The operator can be deployed with Helm v3 chart in /charts as follows:
   * Disable persistence for basic testing. This can be done by adding `--set
     etcd.persistence.enabled=false` to the `helm install` command below.
 
-- Configure the LVM VG and LV names by setting the following values in a local
-  values file or by using `--set` with the `helm install` command below.
-
-    ```
-    operator.nodeSet.spec.lvmPoolVgName=drbdpool # <- Volume Group name of the thick storage pool
-    operator.nodeSet.spec.lvmThinPoolVgName=drbdpool # <- Volume Group name of the thin storage pool
-    operator.nodeSet.spec.lvmThinPoolLvName=thinpool # <- Logical Volume name of the thin pool
-    ```
+- Configure storage pools. By default, helm creates two pools (see [values.yaml]):
+  ```yaml
```

Comment:
Maybe we should put these instructions and this example in their own subsection to avoid the list getting too long and hard to follow.

Reply:
Moved